src/main/java/org/spdx/tools/SpdxToolsHelper.java

```diff
@@ -42,16 +42,16 @@
 
 /**
  * Static helper methods for the various tools
- * 
+ *
  * @author Gary O'Neall
  *
  */
 public class SpdxToolsHelper {
 
 	public enum SerFileType {
-		JSON, RDFXML, XML, XLS, XLSX, YAML, TAG
+		JSON, RDFXML, XML, XLS, XLSX, YAML, TAG, YML
```

Comment:
Rather than add an additional file type, we should just map the .yml file extension to the existing YAML file type.  The SerFileType should represent type type of file (e.g. the format of the content), not the extension used.  This would the change more straight forward.

Reply:
I have done the requested changes but here I have a doubt that if I don't add the YML file type then I will get this error
an enum switch case label must be the unqualified name of an enumeration constant